sdk/src/main/java/com/kin/ecosystem/onboarding/view/OnboardingFragment.kt

```diff
@@ -93,7 +93,7 @@ class OnboardingFragment: Fragment(), IOnboardingView {
                         .setDuration(AnimConsts.Duration.FADE_ANIM)
                         .withStartAction {
                             visibility = View.INVISIBLE
-                            (drawable as AnimatedVectorDrawable).stop()
+                            (drawable as AnimatedVectorDrawableCompat).stop()
```

Comment:
Will come and ask you later what is this drawable and what is the "with" keyword

Reply:
👍